docs/source/create_channel/channel_policies.md

```diff
@@ -0,0 +1,121 @@
+
+Channel policies are important enough that they need to be discussed in their own topic. Unlike other parts of the channel configuration, the policies that govern the channel are determined by how different sections of the `configtx.yaml` file work together. While channel policies can be configured for any use case with few constraints, this topic will focus on how to use the default policies provided by Hyperledger Fabric. By default, Fabric uses a combination of signature policies, ImplicitMeta policies, and Access Control Lists to determine how organizations interact with the channel and agree to update the channel structure. You can learn more about the role of policies in Hyperledger Fabric by visiting the [Policies concept topic](../policies.html).
+
+## Signature policies
+
+Each channel member is associated with a set of signature policies. Signature policies read the digital signatures that are attached to the transactions or channel update requests that are submitted to the channel. Each signature policy lists a set of organizations and roles whose signatures need to be attached to a transaction in order to satisfy the policy. You can see the set of signature policies associated with peer organization Org1 in the **Organizations** section of `configtx.yaml` below:
+```yaml
+- &Org1
+
+  ...
+
+  Policies:
+      Readers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.peer', 'Org1MSP.client')"
+      Writers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.client')"
+      Admins:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin')"
+      Endorsement:
+          Type: Signature
+          Rule: "OR('Org1MSP.peer')"
+```
+
+Org1 is associated with four signature policies. The policies can only be satisfied by signatures from Org1. However, each policy lists a different set of roles from within the organization that satisfy the policy. The `Admins` policy can only be satisfied by the transactions submitted by an identity with an admin role, while only identities with a peer role can satisfy the Endorsement policy. Each transaction can satisfy multiple signature policies. For example, if a transaction submitted to eh channel was endorsed by a peer from Org1 and Org2, the transaction would satisfy the `Endorsement` signature policy that is associated with both Org1 and Org2.
+
+## ImplicitMeta Policies
+
+The signature policies for each organization are read by ImplicitMeta policies at higher levels of the channel configuration. Instead of directly reading the signatures that are submitted to the channel, ImplicitMeta policies read other policies in the channel configuration. An ImplicitMeta policies is satisfied if a transaction satisfies the signature policies that are referenced by that policy.
```

Comment:
The article is about the default policies work, with understanding that once users have the background they need, they can change them. `I added this sentence to the opening: `If you use the default policies used by the Fabric test network or the [Fabric sample configuration](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml), each channel you create will use a combination of signature policies, ImplicitMeta policies, and Access Control Lists to determine how organizations interact with the channel and agree to update the channel structure.`

I also updated the ImplictMeta paragraph in response to your comments: `If your channel uses the default policies, the signature policies for each organization are evaluated by ImplicitMeta policies at higher levels of the channel configuration. Instead of directly evaluating the signatures that are submitted to the channel, ImplicitMeta policies have rules specify a set of other policies in the channel configuration that can satisfy the policy. A transaction can satisfy an ImplicitMeta policy if it can satisfy the underlying set of signature policies that are referenced by the policy.`